dwave_networkx/drawing/pegasus_layout.py

```diff
@@ -69,6 +69,7 @@ def pegasus_layout(G, scale=1., center=None, dim=2, crosses=False):
     >>> pos = dnx.pegasus_layout(G)
 
     """
+    import numpy as np
```

Comment:
Unless there are performance concerns, or circular dependencies, imports should go to the top.

Reply:
Done in commit `2d2485b`